selfdrive/ui/sound.cc

```diff
@@ -13,28 +13,34 @@ extern "C"{
 #include "slplay.h"
 }
 
+int last_volume = 0;
+
 void set_volume(int volume) {
-  char volume_change_cmd[64];
-  sprintf(volume_change_cmd, "service call audio 3 i32 3 i32 %d i32 1 &", volume);
+  if (last_volume != volume) {
```

Comment:
are we sure we're the only ones changing the volume? do the volume buttons do anything?

Reply:
Tested on an EON, volume buttons have no effect